spec/spec_tests/uri_options_spec.rb

```diff
@@ -0,0 +1,123 @@
+require 'lite_spec_helper'
+
+describe 'Uri Options' do
+  include Mongo::ConnectionString
+
+  URI_OPTIONS_TESTS.each do |file|
+
+    spec = Mongo::ConnectionString::Spec.new(file)
+
+    context(spec.description) do
+
+      before(:all) do
+
+        module Mongo
+          class Address
+
+            private
+
+            alias :original_create_resolver :create_resolver
+            def create_resolver(timeout, ssl_options)
+              family = (host == 'localhost') ? ::Socket::AF_INET : ::Socket::AF_UNSPEC
+              info = ::Socket.getaddrinfo(host, nil, family, ::Socket::SOCK_STREAM)
+              FAMILY_MAP[info.first[4]].new(info[3], port, host)
+            end
+          end
+
+          class Server
+
+            # The constructor keeps the same API, but does not instantiate a
+            # monitor and run it.
+            alias :original_initialize :initialize
```

Comment:
This code was removed from master, please update this spec to match what is currently in the connection string spec.

Reply:
Done